.idea/runConfigurations.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this be committed?

No but it also shouldn't be in the project anyways so deal with it

Reply:
I wouldn't